google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -1148,6 +1148,74 @@ def iter_crm_projects(self, parent_type, parent_id):
         for project in resources:
             yield project
 
+    # pylint: disable=inconsistent-return-statements, using-constant-test
```

Comment:
Can you not use pylint disable here? The other methods don't seem to use it.

Reply:
Changing the return type allowed to remove pylint disables. Done.